helm/observability-bundle/templates/apps.yaml

```diff
@@ -46,20 +46,8 @@ spec:
   name: {{ .chartName }}
   namespace: {{ .namespace }}
   version: {{ .version }}
-  {{- if or .extraConfigs (hasSuffix "prometheus-agent" $appName) }}
+  {{- if .extraConfigs }}
   extraConfigs:
-  # If the app is the prometheus agent, we inject of the remote write secret create by the Prometheus Meta Operator
-  {{- if hasSuffix "prometheus-agent" $appName }}
-  - kind: secret
-    {{- if hasPrefix "org-" $.Release.Namespace }}
-    # App is deployed in the org- namespace so the secret name is prefixed by the cluster-id
-    name: {{ printf "%s-remote-write-api-endpoint-config" $.Values.clusterID -}}
-    {{- else }}
-    # App is deployed in the cluster-id namespace so prefix is unneeded
-    name: remote-write-api-endpoint-config
-    {{- end }}
-    namespace: {{ $.Release.Namespace }}
-  {{- end }}
   {{- range $extraConfig := .extraConfigs }}
   - kind: {{ $extraConfig.kind }}
     name: {{ tpl $extraConfig.name $ }}
```

Comment:
Shouldn't you also add the `namespace` field here ? (with `namespace: {{tpl $extraConfig.namespace $}}`)

Reply:
After `kind` and `name`

Reply:
it's the line below :D it's not visible here though

Reply:
Oh sorry my bad !